data/json/snippets/epilogue_factions.json

```diff
@@ -59,13 +95,29 @@
+        "text": "    Melchior's enlightened leadership, combined with practical skills provided by mercenary survivors, allowed Hub-01 to flourish briefly.  It became a mecca for advanced technology, selling off devices and equipment that could no longer be reproduced.  Despite this, Hub-01 was never able to expand sufficiently to renew its own population, nor was it able to unite with any other prosperous community, nor could enough trustworthy mercenaries (or staff) be recruited to replace those who died in its defense.  Hub-01 fell into disrepair and ruin.  After the air processing facility was overtaken by mold, the remaining staff of Hub-01 fled the building and were killed by zombies and bandits.  Melchior's drones - possibly by design - were incapable of maintaining themselves without human hands, and stopped working a few months later.  Without regular maintenance, Melchior's own systems also fell to ruin, and less than a decade after the Cataclysm, humanity's first AI powered down and was forgotten."
+      },
+      {
+        "id": "epilogue_faction_robofac_thrive",
+        "text": "    Melchior's enlightened leadership, backed-up with the muscle of numerous mercenary survivors, allowed Hub-01 to flourish on its' own for a time.  However, while the staff of Hub-01 preferred to ignore the people of the region, the looming shortages of advanced resources and personnel forced them to begin opening up to their neighbors for help.  Fortunately, the knowledge and resources they were able to source from the Exodii, and the influx of fresh recruits from organizations like the Great Library, ensured that the Hub would continue to thrive for the foreseeable future.  While the researchers of the Hub would restrain themselves (or be restrained by Melchior) from engaging in the worst excesses of XEDRA, they continued to explore new ways to survive, and perhaps one day even overcome, the horrors besieging the Earth.  Bionics and mutagens, reality anchors and molecular printing, the future was truly in their hands."
+      },
+      {
+        "id": "epilogue_faction_robofac_thrive_OG",
+        "text": "    While still technically being employees (or property, in Melchior's case) of the federal government, the staff of Hub-01 were initially very reluctant to have anything to do with the Old Guard.  However, the connections and resources of the Old Guard gave them significant leverage in negotiations between the two factions.  After much debate, Hub-01 would eventually vote to rejoin the government in exchange for its' support of the Hub's research, as well as for the legal recognition of Melchior as a fully sentient citizen."
```

Comment:
```suggestion
        "text": "    While still technically being employees (or property, in Melchior's case) of the federal government, the staff of Hub-01 were initially very reluctant to have anything to do with the Old Guard.  However, the connections and resources of the Old Guard gave them significant leverage in negotiations between the two factions.  After much debate, Hub-01 would eventually vote to rejoin the government in exchange for its' support of the Hub's research."
```

Reply:
Melchior was actually only sentient during the events of the two week long portal storm that was the Cataclysm.